index.bs

```diff
@@ -3296,6 +3368,92 @@ failed</dfn> steps given |context| and |navigation status|:
 
 </div>
 
+#### The browsingContext.userPromptClosed Event #### {#event-browsingContext-userPromptClosed}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        BrowsingContextUserPromptClosedEvent = {
+         method: "browsingContext.userPromptClosed",
+         params: UserPromptInfo
+       }
+       UserPromptResult = {
+         context: BrowsingContext,
+         confirmed: bool,
+         ? userInput: text
```

Comment:
```suggestion
         ? userText: text
```

Reply:
I just chose the terminology to match https://chromedevtools.github.io/devtools-protocol/tot/Page/#event-javascriptDialogClosed but I like your suggestion as well.